docs/quantization.md

```diff
@@ -138,37 +120,8 @@ qconfig = config.StaticQuantConfig(calibration_data_reader)  # or qconfig = Dyna
 quantize(model, q_model_path, qconfig)
 ```
 
-2. With Accuracy Aware Tuning
-
-This means user could leverage the advance feature of ONNX Neural Compressor to tune out a best quantized model which has best accuracy and good performance. User should provide `eval_fn`.
-
-``` python
-from onnx_neural_compressor import data_reader
-from onnx_neural_compressor.quantization import tuning, config
-
-class DataReader(data_reader.CalibrationDataReader):
-    def get_next(self): ...
-
-    def rewind(self): ...
-
-
-data_reader = DataReader()
-
-# TuningConfig can accept:
-# 1) a set of candidate configs like tuning.TuningConfig(config_set=[config.RTNConfig(weight_bits=4), config.GPTQConfig(weight_bits=4)])
-# 2) one config with a set of candidate parameters like tuning.TuningConfig(config_set=[config.GPTQConfig(weight_group_size=[32, 64])])
-# 3) our pre-defined config set like tuning.TuningConfig(config_set=config.get_woq_tuning_config())
-custom_tune_config = tuning.TuningConfig(config_set=[config.RTNConfig(weight_bits=4), config.GPTQConfig(weight_bits=4)])
-best_model = tuning.autotune(
-    model_input=model,
-    tune_config=custom_tune_config,
-    eval_fn=eval_fn,
-    calibration_data_reader=data_reader,
-)
-```
-
 ### Specify Quantization Rules
-ONNX Neural Compressor support specify quantization rules by operator name. Users can use `set_local` API of configs to achieve the above purpose by below code:
+Neural Compressor support specify quantization rules by operator name. Users can use `set_local` API of configs to achieve the above purpose by below code:
 
 ```python
 fp32_config = config.GPTQConfig(weight_dtype="fp32")
```

Comment:
all this page use PTQ examples, is it better to use a PTQ example for set_local as well?